ecs.php

```diff
@@ -182,18 +183,16 @@
     $services->set(NoNotOperatorSniff::class);
     $services->set(Psr4Sniff::class);
 
-    // sypmlify rules - see https://github.com/symplify/coding-standard/blob/master/docs/phpcs_fixer_fixers.md
+    // symplify rules - see https://github.com/symplify/coding-standard/blob/master/docs/phpcs_fixer_fixers.md
     // arrays
-    $services->set(ArrayOpenerNewlineFixer::class);
     $services->set(StandaloneLineInMultilineArrayFixer::class);
 
     // annotations
     $services->set(ParamReturnAndVarTagMalformsFixer::class);
 
     // extra spaces
-    $services->set(RemoveSuperfluousDocBlockWhitespaceFixer::class);
```

Comment:
`RemoveSuperfluousDocBlockWhitespaceFixer` and `RemoveSpacingAroundModifierAndConstFixer` don't exist anymore. Eugene used new rules recommended [here](https://github.com/symplify/symplify/issues/2861).